zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1673,6 +1673,16 @@ load_key() {
     return 0
   fi
 
+  # Run early load_key hooks, if they exist
+  if [ -d /libexec/load_key.d ]; then
+    tput clear
+    for _hook in /libexec/load_key.d/*; do
```

Comment:
`_hook` should be declared local at the top of this function.

Reply:
Got it